### PR TITLE
Hotfix/styling and tabbing

### PR DESCRIPTION
**Pull request by Vuurvos1**
Changed form to be more user-friendly, also some overall optimizations and changes for better usability

### CODE REVIEW

File: views/components/chatHeader.ejs [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
...73 6.73 0 01-.9.5 5 5 0 01-1.2.39 6.93 6.93 0 01-1.5.15zM70.42 46a1.42 1.42 0 00.28 0l.32-.07.29-.07a1.18 1.18 0 00.25-.1l.18-.07v-2.57l-1.06.16a2.52 2.52 0 00-1.11.38 1 1 0 00-.37.84v.5a1 1 0 00.14.56.72.72 0 00.41.31 2.22 2.22 0 00.67.13zm14.4 3.26a7.14 7.14 0 01-2.82-.48 2.89 2.89 0 01-1.54-1.64A8.42 8.42 0 0180 44l.16-4.73H78V36l2.36-.78.64-3.9h4v3.93h2.84v4H85V44a3.21 3.21 0 000 .58 1.47 1.47 0 00.17.42.83.83 0 00.3.26 1.08 1.08 0 00.42.11l1.79.19v3.74zm10.37.16a8.27 8.27 0 01-2.44-.35 4.66 4.66 0 01-1.92-1.16 5.27 5.27 0 01-1.25-2.17 11 11 0 01-.44-3.4 13.08 13.08 0 01.4-3.54 5.34 5.34 0 011.18-2.25 4.33 4.33 0 012-1.19 9.51 9.51 0 012.63-.36 16.3 16.3 0 012.54.18 17.59 17.59 0 012.5.63v2.89H97a4 4 0 00-1.73.3 1.67 1.67 0 00-.85 1.07 8.18 8.18 0 00-.23 2.2 7.89 7.89 0 00.22 2.18 1.38 1.38 0 00.83 1 5.22 5.22 0 001.81.24h3.54v2.83A12.66 12.66 0 0199 49a16.16 16.16 0 01-1.91.34 15.76 15.76 0 01-1.9.08zm7.05-.16V29.45h5v7.15a6.3 6.3 0 011.76-1.08 5.54 5.54 0 012.28-.51 4.63 4.63 0 012.6.63 3.34 3.34 0 011.32 1.78 8.58 8.58 0 01.39 2.74v9.1h-5v-8.42a1.43 1.43 0 00-.19-.81 1 1 0 00-.51-.39 2.11 2.11 0 00-.68-.1 2.54 2.54 0 00-.7.11 3.79 3.79 0 00-.68.3 2.71 2.71 0 00-.52.39v8.92z"
```
Comment (SimonPlanje) [inline views/components/chatHeader.ejs:8]: Damn this svg is long. Is this svg cleaned up?

Comment (SimonPlanje) [inline views/components/chatHeader.ejs:8]: Is this svg cleaned up? This is LOOONG

File: views/components/registerForm.ejs [hunk 2]
```diff
@@ -4,45 +4,44 @@
   <form action="/register" method="post">
     <fieldset>
       <label for="name"><b>First name</b></label>
-      <input type="text" name="secondname" id="firstname" />
+      <input tabindex="1" class="textfield" type="text" name="secondname" id="firstname" required />
```
Comment (SimonPlanje) [inline views/components/registerForm.ejs:7]: keep the `type="text"` at first for better readability

File: views/components/chatSidebar.ejs [hunk 3]
```diff
@@ -10,7 +10,7 @@
 
         <li>
             <form class="chatProfile" action="/changeChat" method="post">
-                <button type="submit" name="changeChat" value="<%= data[i]._id %>">
+                <button tabindex="2" type="submit" name="changeChat" value="<%= data[i]._id %>">
```
Comment (SimonPlanje) [inline views/components/chatSidebar.ejs:13]: keep the `type="submit"` at first for better readability

File: views/components/chatSend.ejs [hunk 4]
```diff
@@ -1,6 +1,6 @@
 <form id="messageForm" action="/chat" method="post">
-    <input type="text" id="message" name="message" placeholder="Type a message">
-    <button type="submit" id="sendMessage">
+    <input tabindex="1" type="text" id="message" name="message" placeholder="Type a message">
```
Comment (SimonPlanje) [inline views/components/chatSend.ejs:2]: keep the `type="text"` at first for better readability

File: views/components/chatSend.ejs [hunk 5]
```diff
@@ -1,6 +1,6 @@
...
+    <button tabindex="1" type="submit" id="sendMessage">
```
Comment (SimonPlanje) [inline views/components/chatSend.ejs:3]: keep the `type="submit"` at first for better readability

File: views/components/chatHeader.ejs (hunk 1 above)
Comment (Vuurvos1) [reply views/components/chatHeader.ejs:8]: Yes, it is cleaned up :)

**Pull request merged**